pkgs/development/tools/operator-sdk/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildGoModule, go, lib, fetchFromGitHub, makeWrapper, ... }:
+
+buildGoModule rec {
+  pname = "operator-sdk";
+  version = "0.17.1";
+
+  src = fetchFromGitHub {
+    owner = "operator-framework";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0q360vzqpxjglv6mh427lz4wvqby1sh7gcv04c0ai6ln392sj319";
+  };
+
+  modSha256 = "0mspbnw41c6r3p7kaafza1v10zi4jv0vrnzxwpyh7pkmx150xgvd";
```

Comment:
Sorry for not noticing this before. But since #86376 , you should:

1. Rebase to master.
2. Switch to `vendorModSha256`.